spec/error_reporters/rollbar_spec.rb

```diff
@@ -33,6 +33,18 @@
       ))
     end
 
+    context "given a rack_env with person data" do
+      let(:rack_env) { { "rollbar.person_data" => { email: "foo@bar.com" } } }
```

Comment:
Maybe include `id` and `username` too to document all the things rollbar accepts as shown here: https://rollbar.com/docs/notifier/rollbar-gem/ ? Either that or maybe add an example in the readme.